tests/system/fim/common_tasks/verify_alerts.yml

```diff
@@ -21,7 +21,8 @@
     script: >
       verify_alerts_elasticsearch.py -i {{ item.path }} -e {{ event }} 
       -a {{ hostvars[groups["elasticsearch"][0]]["private_ip"] }}
-      -o {{ missing_alerts_elasticsearch_path }}
+      -o {{ missing_alerts_elasticsearch_path }} 
+      {{ verify_elastic_script_extra_params }}
```

Comment:
Lets standardize this as `elastic_verification_extra_arguments`

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/552/commits/9455e81de250cfb7a9077da377925afe70ae6782.